src/Main.java

```diff
@@ -1,5 +1,70 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
+        task1(); // Задача 1
+        task2(); // Задача 2
+        task3(); // Задача 3
+    }
+
+    //Задача 1
+    public static void task1() {
+        int year = 2019;
+        printYear(year);
+    }
+
+    public static void printYear(int year) {
+        if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)) {
```

Comment:
лишние скобки. У И и так приорите выше, чем у ИЛИ
```suggestion
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```